.github/workflows/build-and-deploy.yml

```diff
@@ -88,6 +88,41 @@ jobs:
           maven-central-password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
           signing-key: ${{ secrets.SIGNING_KEY }}
           signing-password: ${{ secrets.SIGNING_PASSWORD }}
+  # Test jpackage utility
+  test-packaging:
+    needs:
+      - select-java-version
+    strategy:
+      matrix:
+        os: [ windows-2022, macos-12, ubuntu-22.04 ]
+    runs-on: ${{ matrix.os }}
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event_name != 'pull_request'
+        || github.event.pull_request.head.repo.full_name == github.repository
+      )
```

Comment:
why do we need to exclude the forks if no secret is required to package?

Reply:
I've copied it from "test-deploy" without thinking. At first I thought test-deploy was disabled for other reasons, the only downside that comes to my mind is the higher execution time of the pipeline with test-packaging enabled. I'll remove the fork check

Reply:
resolve when done